06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/README.md

```diff
@@ -4,4 +4,4 @@
 - **With**: *independently*
 
 #### Targeted Learning Outcomes
-- [Software Development Environments](https://github.com/andela/learningmap/tree/master/Phase-C/Entry-level%20Developer/Curriculum/35%20-%20Software%20Development%20Environments)
```

Comment:
I know there's no damage however you'll need to be mindful when using git. These simple things can make a spaceship crash. So I'll insist you fix it.